.changes/unreleased/Features-20240405-175733.yaml

```
@@ -0,0 +1,5 @@
+kind: Features
```

Here's a `body` section that would align with the spirit of https://github.com/dbt-labs/dbt-core/issues/9747 :
```suggestion
kind: Features
body: Better error message when trying to select a disabled model
```